practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -450,6 +480,22 @@ private List<String> getLocationIdentifiersByIds(List<Long> locationIds) {
         return locationsIdentifiers;
     }
 
+    private List<IBaseResource> getGroupsAssignedToAPractitioner(Long practitionerId) {
+        SearchParameterMap groupSearchParameterMap = new SearchParameterMap();
+        TokenAndListParam codeListParam = new TokenAndListParam();
+        TokenOrListParam coding = new TokenOrListParam();
+        TokenParam code = new TokenParam();
+        code.setValue("405623001");
+        code.setSystem("http://snomed.info/sct");
+        coding.add(code);
+        codeListParam.addAnd(coding);
+        groupSearchParameterMap.add(CODE, codeListParam);
+        IBundleProvider groupsBundle = groupIFhirResourceDao.search(groupSearchParameterMap);
+        return groupsBundle != null
+                ? groupsBundle.getResources(0, groupsBundle.size())
+                : new ArrayList<>();
+    }
```

Comment:
@rehammuzzamil The `practitionerId` is also not used here. The code below does not account for the Practitioner Id on the search parameters.

Reply:
Updated